### PR TITLE
Update Sepolia Gas Config 

### PR DESCRIPTION
**Pull request by stevieraykatz**
This PR mirrors the expected outcome of https://github.com/base-org/contract-deployments/pull/134. The implementation is significantly simpler since the owner of SystemConfig is an EOA and not a multisig.

### CODE REVIEW

**Comment by stevieraykatz** [comment]
Omitted rollback functionality, we can make that call if necessary using this same script and just changing the env var

File: sepolia/2024-03-07-ecotone-sysconfig-updates/script/UpdateGasConfig.sol [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    address internal L1_SYSTEM_CONFIG = vm.envAddress("L1_SYSTEM_CONFIG_ADDRESS");
+    uint256 internal SCALAR = vm.envUint("SCALAR");
+
+    function _postCheck() internal view {
+        require(SystemConfig(L1_SYSTEM_CONFIG).scalar() == SCALAR);
+        require(SystemConfig(L1_SYSTEM_CONFIG).overhead() == 0);
+    }
+
+    function run() public {
+        vm.startBroadcast();
```
Comment (anikaraghu) [inline sepolia/2024-03-07-ecotone-sysconfig-updates/script/UpdateGasConfig.sol:18]: nit: could be good to provide a the address here to be more explicit

File: sepolia/2024-03-07-ecotone-sysconfig-updates/Makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+ifndef LEDGER_ACCOUNT
+override LEDGER_ACCOUNT = 0
+endif
+
+# Unpause OptimismPortal Commands
+
+.PHONY: update-gas-config
+update-gas-config:
+	forge script --rpc-url $(L1_RPC_URL) UpdateGasConfigSepolia --ledger --hd-paths "m/44'/60'/$(LEDGER_ACCOUNT)'/0/0" --broadcast
```
Comment (anikaraghu) [inline sepolia/2024-03-07-ecotone-sysconfig-updates/Makefile:13]: I think it's just an EOA at the moment, so we'll need to do `--private-key` vs the `--ledger --hd-paths ` etc

Comment (stevieraykatz) [reply sepolia/2024-03-07-ecotone-sysconfig-updates/Makefile:13]: I think this should work also? From this issue: https://github.com/foundry-rs/foundry/issues/2709 

```
forge script script/Deploy.s.sol -f $ETH_RPC_URL -vvvv --ledger --hd-paths "m/44'/60'/0'/2" --legacy --broadcast --verify --sender "0xABCD"
```

Comment (mdehoog) [reply sepolia/2024-03-07-ecotone-sysconfig-updates/Makefile:13]: i think we should update it to use a private key envvar, like https://github.com/base-org/contract-deployments/blob/6fb47e258d2252022bf3da1c0e6f183de9c39a86/sepolia/2024-02-28-pause-unpause-portal/Makefile#L19-L20

Comment (stevieraykatz) [reply sepolia/2024-03-07-ecotone-sysconfig-updates/Makefile:13]: Ok. Will update.

**Review by mdehoog — APPROVED** [review]

**Pull request merged**